### PR TITLE
Translate concepts/services-networking/dual-stack into Japanese

### LINKED ISSUE #22908: Translate concepts/services-networking/dual-stack into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [concepts/services-networking/dual-stack](https://kubernetes.io/docs/concepts/services-networking/dual-stack) into Japanese.

**Why is this needed**

This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by shuuji3**
Closes #22908.

### CODE REVIEW

**Comment by shuuji3** [comment]
Preview: https://deploy-preview-22909--kubernetes-io-master-staging.netlify.app/ja/docs/concepts/services-networking/dual-stack/

File: content/ja/docs/concepts/services-networking/dual-stack.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+`.spec.ipFamily`は、次のいずれかに設定できます。
+
+   * `IPv4`: APIサーバーは`ipv4`の`service-cluster-ip-range`の範囲からIPアドレスを割り当てます
+   * `IPv6`: APIサーバーは`ipv6`の`service-cluster-ip-range`の範囲からIPアドレスを割り当てます
+
+次のServiceのspecには`ipFamily`フィールドが含まれていません。Kubernetesは、最初に設定した`service-cluster-ip-range`の範囲からこのServiceにIPアドレス(別名「cluster IP」)を割り当てます。
+
+{{< codenew file="service/networking/dual-stack-default-svc.yaml" >}}
+
+次のServiceのspecには`isFamily`フィールドが含まれています。Kubernetesは、最初に設定した`service-cluster-ip-range`の範囲からこのServiceにIPv6のアドレス(別名「cluster IP」)を割り当てます。
```
Comment (oke-py) [inline content/ja/docs/concepts/services-networking/dual-stack.md:78]:
```suggestion
次のServiceのspecには`ipFamily`フィールドが含まれています。Kubernetesは、最初に設定した`service-cluster-ip-range`の範囲からこのServiceにIPv6のアドレス(別名「cluster IP」)を割り当てます。
```

**Pull request merged**